packages/eas-cli/src/project/publish.ts

```diff
@@ -140,11 +142,61 @@ export async function buildUpdateInfoGroupAsync(assets: CollectedAssets): Promis
   return updateInfoGroup as UpdateInfoGroup;
 }
 
+export async function buildBundlesAsync({
+  projectDir,
+  inputDir,
+}: {
+  projectDir: string;
+  inputDir: string;
+}) {
+  const packageJSON = JsonFile.read(path.resolve(projectDir, 'package.json'));
+  if (!packageJSON) {
+    throw new Error('Could not locate package.json');
+  }
+  const isCLIInstalled =
+    !!(packageJSON.dependencies as JSONObject)['expo-cli'] ||
+    !!(packageJSON.devDependencies as JSONObject)['expo-cli'];
+  if (!isCLIInstalled) {
```

Comment:
Just realized this won't work correctly for people who don't have Expo CLI as a local dependency. (We recommend installing it globally, so most people have it installed globally.)

Reply:
oh great catch. Lemme think about how to fix that.